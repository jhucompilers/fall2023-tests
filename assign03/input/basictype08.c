int main(void) {
  volatile char vc;
  volatile unsigned short vus;
  volatile long vl;

  vc = 1;
  vus = 2;
  vl = 3;

  return 0;
}
