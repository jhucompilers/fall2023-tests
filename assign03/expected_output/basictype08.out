0|main|function|function () returning int
2|vc|variable|volatile char
2|vus|variable|volatile unsigned short
2|vl|variable|volatile long
